doc/plugins/limit-req.md

```diff
@@ -104,6 +106,78 @@ Server: APISIX web server
 
 This means that the limit req plugin is in effect.
 
+### How to enable on the `consumer`
+
+To enable the `limit-req` plugin on the consumer, it needs to be used together with the authorization plugin. Here, the key-auth authorization plugin is taken as an example.
+
+1. Bind the `limit-req` plugin to the consumer
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/consumers -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "username": "consumer_jack",
+    "plugins": {
+        "key-auth": {
+            "key": "auth-jack"
+        },
+        "limit-req": {
+            "rate": 1,
+            "burst": 1,
+            "rejected_code": 403,
+            "key": "consumer_name"
```

Comment:
When this plugin is binding to the consumer object, it can be handled automatically, the declaration here is redundant

Reply:
I think it is necessary, because the value of `consumer_name` needs to be determined according to the value of `key`.

Reply:
the `key` is always `consumer_name` when binding with consumer, right?

Reply:
No, the 'key' enumeration type has no default value.

Reply:
> No, the 'key' enumeration type has no default value.

Does this answer have anything to do with my question?

Reply:
Sorry, my response is not clear. When bound to the `consumer`, the `key` is optional, and when the `key` is other values (for example:`remote_addr`), it can be used normally.

Reply:
got it